tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -177,43 +177,8 @@ partial class Build
             }
         });
 
-    Target BuildAndRunProfilerLinuxIntegrationTests => _ => _
-        .Requires(() => IsLinux && !IsArm64)
-        .After(BuildTracerHome, BuildProfilerHome, BuildNativeLoader, ZipMonitoringHome)
-        .Description("Builds and runs the profiler linux integration tests.")
-        .DependsOn(BuildProfilerLinuxIntegrationTests)
-        .DependsOn(RunProfilerLinuxIntegrationTests);
-
-    Target BuildProfilerLinuxIntegrationTests => _ => _
-        .Description("Builds the profiler linux integration tests.")
-        .Requires(() => IsLinux && !IsArm64)
-        .DependsOn(CompileProfilerSamplesLinux)
-        .DependsOn(CompileProfilerLinuxIntegrationTests);
-
-    Target CompileProfilerLinuxIntegrationTests => _ => _
-        .Unlisted()
-        .After(PublishProfilerLinux)
-        .After(CompileProfilerSamplesLinux)
-        .Executes(() =>
-        {
-            // Build the actual integration test projects for x64
-            var integrationTestProjects = ProfilerDirectory.GlobFiles("test/*.IntegrationTests/*.csproj");
-            DotNetBuild(x => x
-                    // .EnableNoRestore()
-                    .EnableNoDependencies()
-                    .SetConfiguration(BuildConfiguration)
-                    // .SetTargetPlatform(MSBuildTargetPlatform.x64)
-                    .SetNoWarnDotNetCore3()
-                    .When(TestAllPackageVersions, o => o.SetProperty("TestAllPackageVersions", "true"))
-                    .When(IncludeMinorPackageVersions, o => o.SetProperty("IncludeMinorPackageVersions", "true"))
-                    .When(!string.IsNullOrEmpty(NugetPackageDirectory), o =>
-                        o.SetPackageDirectory(NugetPackageDirectory))
```

Comment:
I think you probably want this line in `BuildAndRunProfilerIntegrationTests` too, so we use the same location for the NuGet packages.

Reply:
When I added it, the dotnet stop with an error `Unknown switch: --package`